server/form-pages/apply/access-and-healthcare/accessNeeds.ts

```diff
@@ -63,7 +63,7 @@ export default class AccessNeeds implements TasklistPage {
   }
 
   next() {
-    return ''
+    return this.body.additionalNeeds.find(need => need === 'mobility') ? 'access-needs-mobility' : ''
```

Comment:
Could we use `includes` instead?:

```suggestion
    return this.body.additionalNeeds.includes('mobility') ? 'access-needs-mobility' : ''
```